package.json

```diff
@@ -59,7 +63,7 @@
     "flow-bin": "0.54.1",
     "flow-runtime": "0.14.0",
     "lazysizes": "^4.1.6",
-    "lodash": "4.17.13",
+    "lodash": "4.17.15",
```

Comment:
Another lib that we will want to keep update to date regularly. This one had the most violations because it is so common. Knowing this, we will need to keep our eyes on libs like this and ensure that we are beginning to see less alerts from GH and `yarn audit` as time goes by.